php/containers-schema.json

```diff
@@ -170,6 +170,31 @@
 							"pattern": "^/[a-z/_0-9-:]+$"
 						}
 					},
+					"caddy_routes": {
+						"type": "array",
+						"items": {
+							"type": "object",
+							"additionalProperties": false,
+							"minProperties": 4,
+							"properties": {
+								"route": {
+									"type": "string",
+									"pattern": "^(/[a-z-]+)*$"
+								},
+								"sub_domain": {
+									"type": "string",
+									"pattern": "^([a-z-]*\\.)*$"
+								},
```

Comment:
I'd for for now like to not have this in here since it is only necessary for community containers. we already have https://github.com/szaimen/aio-caddy for that...

Reply:
46d702d